main_routers/system_router.py

```
@@ -966,6 +968,111 @@ def _is_similar_to_recent_proactive_chat(lanlan_name: str, message: str) -> tupl
+                    f"http://127.0.0.1:{MEMORY_SERVER_PORT}/cache/{lanlan_name}",
+                    json={"input_history": json.dumps(memory_payload, ensure_ascii=False)},
+                    timeout=5.0,
+                )
+                if not memory_response.is_success:
+                    memory_cached = False
+                    logger.warning(
+                        f"[{lanlan_name}] 弱化版搭话写入 recent memory 失败 "
+                        f"turn_id={turn_id} status={memory_response.status_code} "
+                        f"body={memory_response.text[:500]}"
+                    )
+        except Exception as memory_error:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)